tests/rustdoc-gui/search-result-color.goml

```diff
@@ -65,6 +65,12 @@ assert-css: (
     {"border-bottom-color": "#aaa3"}
 )
 
+assert-css: (
```

Comment:
Isn't it already what I did with the `ALL` modifier ?

Reply:
`ALL` means all matching elements. But you only test it in one theme. Moving it into the function would make it tested on all themes since this function is called for all themes.

Reply:
Done hopefully :)